src/BlazorBoilerplate.CommonUI/Pages/DbLogViewer.razor

```diff
@@ -0,0 +1,122 @@
+@inject HttpClient Http
+@inject IMatToaster matToaster
+@using BlazorBoilerplate.Shared.DataModels
+@using static Microsoft.AspNetCore.Http.StatusCodes
+
+<h1>Serilog MSSqlDb Log Viewer</h1>
+<p>
+    Secured Page.<br />
+    Paginated viewing and fetching of system logs from the database.
+</p>
+
+@if (apiResponses == null)
+{
+    <LoadingBackground ShowLogoBox="true">
+        <label>Loading SQL Log Data</label>
+    </LoadingBackground>
+}
+else
+{
+    <div class="row">
+        <div class="font-weight-bold">Refresh Timer:</div>
+        <RefreshTimer ElapsedEvent="@(async (_, __) => await LoadData())" />
+    </div>
+    <MatPaginator PageSize="@pageSize" Page="@(async a=> await OnPage(a))" Length="@logCountTotal">
+
+    </MatPaginator>
+    <MatTable Items="@dBLogItems" Class="mat-elevation-z5" Striped="true" PageSize="@pageSize" ShowPaging="false">
+        <MatTableHeader>
+            <th><div style="max-width:165px;">@*<MatSlideToggle Value="@timer.Enabled" ValueChanged="@(a=>UpdateTimer(a))" TValue="bool" Style="margin-right:16px;"></MatSlideToggle>*@ Timestamp</div></th>
+            <th>Logging Level</th>
+            <th>Log Message</th>
+            <th>Exception (if any)</th>
+            <th>Recorded Properties</th>
+        </MatTableHeader>
+        <MatTableRow>
+            <td><span style="font-size:small">@context.TimeStamp.ToString()</span></td>
+            <td><span style="font-size:small">@context.Level</span></td>
+            <td><span style="font-size:small">@context.Message</span></td>
+            <td><span style="font-size:small">@context.Exception?.ToString()</span></td>
+            <td>
+                @if (context.LogProperties?.Any() ?? false)
+                    {
+                    <ul>
+                        @foreach (var propertyPair in context.LogProperties)
+                            {
+                            <li style="font-size:small">@($"{propertyPair.Key}:  {propertyPair.Value}")</li>
+                            }
+                    
+                    </ul>
+                    }
+              
+            </td>
+        </MatTableRow>
+    </MatTable>
+
+}
+
+
+@code {
+    ApiResponseDto[] apiResponses = new ApiResponseDto[] { };
+    List<DbLog> dBLogItems = new List<DbLog>();
+
+    private int pageSize { get; set; } = 10;
+    private int pageIndex { get; set; } = 0;
+    private int logCountTotal { get; set; } = 0;
+
+    protected override async Task OnInitializedAsync()
+    {
+        await LoadData();
+   
+    }
+    async Task OnPage(MatPaginatorPageEvent e)
+    {
+        pageSize = e.PageSize;
+        pageIndex = e.PageIndex;
+
+        await LoadData().ConfigureAwait(true);
+
+    }
+    protected async Task LoadData()
+    {
+        ApiResponseDto apiResponse = null;
+        try
+        {
+            apiResponse = await Http.GetJsonAsync<ApiResponseDto>($"api/DbLog?page={this.pageIndex}&pageSize={this.pageSize}");
```

Comment:
@np-at  please update this for the latest System.Net.Http.Json:

https://devblogs.microsoft.com/aspnet/blazor-webassembly-3-2-0-preview-3-release-now-available/?


Microsoft.AspNetCore.Blazor.HttpClient | System.Net.Http.Json
-- | --
GetJsonAsync | GetFromJsonAsync
PostJsonAsync | PostAsJsonAsync
PutJsonAsync | PutAsJsonAsync

Reply:
I have just tested: you can use one version or the other. The Post and Put are a little different, so in my last PR I have preferred Post and Put without "As".
In fact I was interested to solution of https://github.com/dotnet/runtime/issues/31553#event-3170368286
but I still experience this issue with GetFromJsonAsync.
I am still testing.